src/main/java/bizbook/logic/commands/DeleteCommand.java

```diff
@@ -47,10 +47,13 @@ public CommandResult execute(Model model) throws CommandException {
 
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deletePerson(personToDelete);
-        model.getFocusedPerson().set(null);
+        Person focusedPerson = model.getFocusedPerson().get();
+        // Need to clear the person from the contact details panel if we are deleting them
+        if (focusedPerson != null && focusedPerson.equals(personToDelete)) {
```

Comment:
```suggestion
        if (personToDelete.equals(focusedPerson)) {
```
Since person class has a null check in its equals function, this check can be simplified.